http/server.ts

```diff
@@ -540,7 +540,10 @@ function hostnameForDisplay(hostname: string) {
   return hostname === "0.0.0.0" ? "localhost" : hostname;
 }
 
-/** Serves HTTP requests with the given handler.
+/**
+ * @deprecated (will be removed after 0.196.0) Use `Deno.serve` instead.
```

Comment:
Discussed this in the core team. We think this is too common to remove soon. Can you update this to `will be removed at 1.0.0`? (Note: We expect we would release std@1.0.0 this year)

Reply:
Sure, I'll change this now.